scripts/utils.js

```diff
@@ -28,3 +30,48 @@ export function validJSON(string) {
 export function cleanRecord(record) {
   return omit(record, ["id", "schema", "last_modified"]);
 }
+
+export function recordField(displayField, record) {
+  if (record.hasOwnProperty(displayField)) {
+    const field = record[displayField];
+    if (typeof field === "string") {
+      return field;
+    } else if (typeof field === "object") {
+      return JSON.stringify(field);
+    } else {
+      return String(field);
+    }
+  } else if (displayField === "__json") {
+    return <code>{JSON.stringify(cleanRecord(record))}</code>;
+  } else if (displayField.indexOf(".") !== -1) {
+    const fields = displayField.split(".");
+
+    if (record.hasOwnProperty(fields[0])) {
+      return recordField(fields.splice(1).join("."), record[fields[0]]);
```

Comment:
Also be careful as `splice` mutates field; it's no big deal here but I'd rather see `slice` used instead.